src/components/FilterPanel/index.tsx

```diff
@@ -152,7 +135,7 @@ class FilterPanel extends React.Component<Props, {}> {
       availableValues,
       onSubmit
     } = this.props;
-    const entityName = attrTabValue[selectedEntity];
+    const entityName = selectedEntity.replace(/_/gi, ' ').slice(0, -1);
```

Comment:
Why are we doing this?

Reply:
![baker](https://user-images.githubusercontent.com/27675170/58348019-68e55980-7e2d-11e9-9312-963fe1631866.png)

Reply:
shouldn't that come from `displayName`?